app/controllers/mailing_list/steps_controller.rb

```diff
@@ -42,7 +42,7 @@ def crm_store
     def resolve_layout
       is_first_page = @current_step.instance_of? MailingList::Steps::Name
 
-      return "registration_attractive" if is_first_page
+      return "registration_with_side_images" if is_first_page
```

Comment:
I've removed `registration_attractive` as part of #1925 so I think this will conflict

Reply:
I think it's ok. The 'attractive' one [is the one with photos at each side](https://user-images.githubusercontent.com/128088/139671307-46a15ed7-5986-4a39-a3de-f27c954ed3b4.png) - is #1925 intending to remove the images from the first page of the regular mailing list signup too?